apps/myjobs/config/configuration_singleton.rb

```diff
@@ -58,6 +58,11 @@ def show_job_options_account_field?
     to_bool(ENV.fetch('OOD_SHOW_JOB_OPTIONS_ACCOUNT_FIELD', true))
   end
 
+  # Override variable for (not) showing job array in the job composer
+  def hide_job_arrays?
+    to_bool(ENV.fetch('OOD_HIDE_JOB_ARRAYS', true))
```

Comment:
I think this should default to false to preserve existing behavior. 

```suggestion
    to_bool(ENV.fetch('OOD_HIDE_JOB_ARRAYS', false))
```

Reply:
ah yes you're right, my apologies.

Reply:
No problem at all!